File: problemas/1000/1046.py
(horaInicio, horaFim) = map(int, input().split(' '))
duracao = horaFim - horaInicio if horaFim > horaInicio else 24 - horaInicio + horaFim
print(f'O JOGO DUROU {duracao} HORA(S)')

File: problemas/1000/1046.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final horaInicio = int.parse(entrada[0]);
  final horaFim = int.parse(entrada[1]);

  final duracao = horaFim > horaInicio
    ? horaFim - horaInicio
    : 24 - horaInicio + horaFim;

  stdout.writeln('O JOGO DUROU $duracao HORA(S)');
}